WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -383,7 +383,7 @@ public static void registerForCloudMessaging(Context context) {
      * select the first one.
      */
     public static Blog getCurrentBlog() {
-        if (currentBlog == null || !wpDB.isDotComBlogVisible(currentBlog.getRemoteBlogId())) {
+        if (currentBlog == null) {
```

Comment:
Right, it feels a bit risky. We could make the blog visible if the user search/select it.